coherence/src/main/java/io/micronaut/coherence/InjectorImpl.java

```diff
@@ -36,9 +36,8 @@ public class InjectorImpl implements Injector {
     public void inject(Object target) {
         ApplicationContext ctx = CoherenceContext.getApplicationContext();
         BeanDefinition bd = ctx.getBeanDefinition(target.getClass());
-        bd.inject(ctx, target);
```

Comment:
why do we remove `bd.inject(txt, target);`?

Reply:
From Gitter:

> Tim Yates: What's the replacement for BeanDefinition.inject in Micronaut 4? [Coherence does this](https://github.com/micronaut-projects/micronaut-coherence/blob/91a454fb6c7943fb0eafb7c107eff56b107f39d0/coherence/src/main/java/io/micronaut/coherence/InjectorImpl.java#L39)
> 
> Graeme Rocher: 
> kind of weird they are doing that
> seems they are injecting the bean twice
> once on line 39 and another time on line 43
> imo line 39 should just be removed